app/assets/stylesheets/views/_specialist-document.scss

```diff
@@ -1,3 +1,33 @@
+$contents-link-background: #dee0e2;
+
 .specialist-document {
   @include sidebar-with-body;
 }
+
+.govuk-sticky-element {
+  width: 100%;
+  margin: 0 $gutter-half;
+}
+
+.contents-list-link-wrapper {
+  background-color: $contents-link-background;
```

Comment:
Need to look at restructuring these styles for when JS is disabled. At the moment it looks like this: 

![screen shot 2017-11-24 at 08 59 48](https://user-images.githubusercontent.com/861310/33202876-f7d45c18-d0f5-11e7-92b5-441f47337ab3.png)

I don't think it needs the background colour, width or padding in this situation.